src/GitHub.App/GlobalSuppressions.cs

```diff
@@ -6,6 +6,8 @@
 [assembly: SuppressMessage("Microsoft.Naming", "CA1703:ResourceStringsShouldBeSpelledCorrectly", MessageId = "Git", Scope = "resource", Target = "GitHub.App.Resources.resources")]
 [assembly: SuppressMessage("Microsoft.Globalization", "CA1305:SpecifyIFormatProvider", MessageId = "System.String.Format(System.String,System.Object,System.Object,System.Object)", Scope = "member", Target = "GitHub.Services.PullRequestService.#CreateTempFile(System.String,System.String,System.String)")]
 [assembly: SuppressMessage("Microsoft.Globalization", "CA1305:SpecifyIFormatProvider", MessageId = "System.String.Format(System.String,System.Object,System.Object,System.Object)", Scope = "member", Target = "GitHub.Services.PullRequestService.#CreateTempFile(System.String,System.String,System.String,System.Text.Encoding)")]
+[assembly: SuppressMessage("Reliability", "CA2007:Do not directly await a Task", Justification = "We always need to think about what thread we're running on")]
```

Comment:
This justification seems wrong - if we need to think about what thread we're running on, then we _should_ be warning when awaiting a `Task` - if you don't add a call to `.ConfigureAwait()` then you're implicitly requesting `.ConfigureAwait(true)` which isn't correct in all cases.

I think my preference here would be to leave the warnings in, and go through and address each warning individually. That's a lot of work, but I think it's the only way we can choose the correct behavior for each case.